toolchains/lowrisc_toolchain_rv32imc/lowrisc_toolchain_rv32imc.bzl

```diff
@@ -238,7 +246,7 @@ def compiler_components(system_hdr_deps, injected_hdr_deps):
     native.filegroup(
         name = "compiler_components",
         srcs = [
-            "//toolchains/lowrisc_toolchain_rv32imc/gcc_wrappers:all",
+            "//toolchains/lowrisc_toolchain_rv32imc/clang_wrappers:all",
```

Comment:
We should consider branching this entire file into a new file so we have separate files for configuring gcc vs. clang.  I think we could then make the compiler selection in opentitan's workspace file by just invoking the appropriate "register toolchain" call.  (we may also need some selection logic in this subdir's BUILD file as well).

Reply:
To keep things nice and clear and avoid the notion of multiple toolchains inside something we also call a toolchain, I'd like to have users invoke the lowrisc gcc toolchain by using the lowrisc gcc_only releases.

Reply:
I don't have a strong opinion on this.  We ought to download whichever archive is agreed upon by the TC as "the approved toolchain".

Reply:
SGTM, I can readd a file that looks like the old one too.